public/javascripts/scrs.js

```diff
@@ -7,7 +7,6 @@ $(document).ready($(function() {
     var notYetToggle = $("#notPlanningToYet-hidden");
 
     var director = $("#completionCapacity-director");
-    var company_secretary = $("#completionCapacity-company_secretary");
```

Comment:
This is reverting the commit that was merged today...You'll need to back out the change to JS and back out the rename from the commit.